vcpkg/ports/qgis/portfile.cmake

```
@@ -48,6 +47,7 @@ list(APPEND QGIS_OPTIONS "-DFLEX_EXECUTABLE=${FLEX}")
 # let's keep things clean and tidy and put them at a predictable location
 list(APPEND QGIS_OPTIONS "-DQGIS_INCLUDE_SUBDIR=include/qgis")
 list(APPEND QGIS_OPTIONS "-DBUILD_WITH_QT6=ON")
+list(APPEND QGIS_OPTIONS "-DUSE_CCACHE:BOOL:=OFF")
```

```suggestion
list(APPEND QGIS_OPTIONS "-DUSE_CCACHE:BOOL:=OFF")
list(APPEND QGIS_OPTIONS "-DUNITY_BUILD=ON")
```

This should speed up things

```suggestion
list(APPEND QGIS_OPTIONS "-DUSE_CCACHE=OFF")
```

this `:BOOL:` looks weird to me

---

Re :BOOL: , that's used throughout this file, when should we use it vs not use it?